class-tgm-plugin-activation.php

```diff
@@ -814,11 +818,19 @@ public function notices() {
 					}
 					unset( $plugin_slug );
 
-					$count       = count( $plugin_group );
-					$last_plugin = array_pop( $linked_plugins ); // Pop off last name to prep for readability.
-					$imploded    = empty( $linked_plugins ) ? '<em>' . $last_plugin . '</em>' : '<em>' . ( implode( ', ', $linked_plugins ) . '</em> ' . esc_html__( 'and', 'tgmpa' ) . ' <em>' . $last_plugin . '</em>' );
+					$count          = count( $plugin_group );
+					$linked_plugins = array_map( array( 'TGM_Utils', 'wrap_in_em' ), $linked_plugins );
+					$last_plugin    = array_pop( $linked_plugins ); // Pop off last name to prep for readability.
+					$imploded       = empty( $linked_plugins ) ? $last_plugin : ( implode( ', ', $linked_plugins ) . ' ' . esc_html_x( 'and', 'plugin A *and* plugin B', 'tgmpa' ) . ' ' . $last_plugin );
 
-					$rendered .= '<p>' . sprintf( translate_nooped_plural( $this->strings[ $type ], $count, 'tgmpa' ), $imploded, $count ) . '</p>';
+					$rendered .= sprintf(
+						$line_template,
+						sprintf(
```

Comment:
If we'd go that way, we'd have to do that for quite a few strings I think - would be more appropriate in a method which reads in all the strings. Then again - what to do if a custom string or a translation doesn't have the right nr of placeholders ? Fall back to our default in English ? That could make for some strange mixed language pages, including mixed terminology (plugin/theme requires .. etc).
Let leave it to the configurators and translators for now until someone reports an issue with it.